template/script.sh.erb

```diff
@@ -19,10 +19,11 @@ setup_env () {
   # the host into the guest, and so those values may vary between sites.
 
   module load ondemand
-  module load xalt/latest <%= context.version %>
```

Comment:
Might be good to keep XALT as one of first things.  No harm in having numerous `module load` that only load a single module.  I think there are environment variables that XALT may set that could impact container things but I think that may only matter as long as XALT is loaded before container is executed so maybe a non-issue.

Reply:
Done, moved it to the first `module load ondemand` statement.